### PR TITLE
Fix query dropdown inconsistencies

### PR DESCRIPTION
**Pull request by oisincoveney**
Adds a few small fixes to the query dropdown.

1. Adds a nicer back button for breakout
image

2. Changes the behavior of the filter and summary dropdowns to ensure that we always go back to the list when a back button is clicked

### CODE REVIEW

**Comment by oisincoveney** [comment]
This change is part of the following stack:

- #50152
    - #50644 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

File: enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx [hunk 1]
```diff
@@ -25,18 +28,35 @@ export const BreakoutPickerInner = ({
...
-      onUpdateBreakoutColumn={(_, column) =>
-        breakoutItem?.updateBreakout(column)
-      }
-      onClose={() => onClose?.()}
-      isMetric={false}
-    />
+    <Stack spacing={0} w="18.75rem">
+      <Button
+        py="md"
+        px="md"
```
Comment (heypoom) [inline enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx:34]:
```suggestion
        p="md"
```

File: enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx [hunk 2]
```diff
@@ -25,18 +28,35 @@ export const BreakoutPickerInner = ({
...
-      breakout={breakoutItem?.breakout}
-      breakoutIndex={breakoutItem?.breakoutIndex}
-      onAddBreakout={onAddBreakout}
-      onUpdateBreakoutColumn={(_, column) =>
-        breakoutItem?.updateBreakout(column)
-      }
-      onClose={() => onClose?.()}
-      isMetric={false}
-    />
+    <Stack spacing={0} w="18.75rem">
```
Comment (heypoom) [inline enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx:31]: I wonder if this 300px width should be overrideable via a className prop?

**Review by heypoom — APPROVED** [review]
Tested locally, works nicely.

File: enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx (hunk 2 above)
Comment (oisincoveney) [reply enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx:31]: Let's leave it for now and add it as a polish thing

**Pull request merged**